cdk/lib/security-hq.ts

```diff
@@ -69,6 +71,18 @@ export class SecurityHQ extends GuStack {
       }
     );
 
+    const auditDataS3BucketName = new GuStringParameter(
+      this,
+      'AuditDataS3BucketName',
+      {
+        description:
+          'Name of the S3 bucket to fetch auditable data from (e.g. Janus data)',
+        default: `/${this.stack}/${SecurityHQ.app.app}/audit-data-s3-bucket/name`,
+        fromSSM: true,
+      }
+    );
+    const auditDataS3BucketPath = [this.stack, this.stage, '*'].join('/');
```

Comment:
Very minor point, but I'm wondering if could this be string interpolation like on line 80 instead of this array join?

Reply:
Yeah, good point - I was looking at an example elsewhere when I wrote this 😆